app/src/main/java/com/futurumgame/base/factories/Factory.java

```diff
@@ -43,17 +43,17 @@ public final int getLevel() {
         return level;
     }
 
-    public final Units getCapacity(){
+    public final Units getCapacity() {
         return internalCapacity;
     }
 
-    public final Units getStorage(){
+    public final Units getStorage() {
         return internalStorage;
     }
 
     public final void workManually() {
         T resource = work();
-        Units multiplier = new Units(5, Math.floor(1/Math.log10((level+100)/169.0 +1)-3));
+        Units multiplier = new Units(5, Math.floor(1 / Math.log10((level + 100) / 169.0 + 1) - 3));
```

Comment:
Make the Math.floor ... into a separate private method. It's not immediately clear what this does.